spec/generators/rollbar/rollbar_generator_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-describe :rollbar do
+describe 'generator', :rollbar do
```

Comment:
@joneslee85 tests are failing cause this, https://github.com/rails/rails/blob/v3.0.20/railties/lib/rails/generators.rb#L128, cannot load the generator.

genspec can however load the generator using the class constant. If you change this line with the next specs should pass:

```ruby
require 'generators/rollbar/rollbar_generator'

describe Rollbar::Generators::RollbarGenerator do
```